content/src/content/jcr_root/apps/core/wcm/sandbox/components/tabs/v1/tabs/_cq_dialog/.content.xml

```diff
@@ -55,6 +56,38 @@
                             </columns>
                         </items>
                     </tabitems>
+                    <options
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Options"
+                        sling:resourceType="granite/ui/components/coral/foundation/container"
+                        margin="{Boolean}true">
+                        <items jcr:primaryType="nt:unstructured">
+                            <columns
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns"
+                                margin="{Boolean}true">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <column
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/container">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <activeItem
```

Comment:
@vladbailescu - why do we need the hidden field, can't we just use the value of the select?

Reply:
The select is populated (and updated) after the dialog is loaded based on the items in the children editor. It would not pickup the defined value if we did not store that somewhere. I ended up using a hidden field although I initially thought of using a data attribute but Granite EL does not offer access to form values so I could not set the current value on that attribute.

Reply:
Gotcha, we could provide a datasource for the select to initially populate it, but might be overkill.

Reply:
It would still need to be updated from JS for uncommitted changes in dialog. So we would have two ways of setting the values, seems quite difficult to maintain.